pkg/kubelet/kubelet_node_status.go

```diff
@@ -426,9 +426,16 @@ func (kl *Kubelet) setNodeAddress(node *api.Node) error {
 			// We tried everything we could, but the IP address wasn't fetchable; error out
 			return fmt.Errorf("can't get ip address of node %s. error: %v", node.Name, err)
 		} else {
+			var nodeInternalIP string
```

Comment:
how is this different from specifying the nodeIP above? this seems like a roundabout way to set this